packages/components/src/components/SquiggleViewer/utils.ts

```diff
@@ -3,9 +3,11 @@ import {
   PartialPlaygroundSettings,
   PlaygroundSettings,
 } from "../PlaygroundSettings.js";
+import { CalculatorState } from "../Calculator/calculatorReducer.js";
 
 export type LocalItemSettings = {
   collapsed: boolean;
+  calculator: null | CalculatorState;
```

Comment:
Note this change. I wasn't sure where to put it, this seemed decent enough. We could also make a new set of states called ``LocalItemCalculators``, if that might be better.

Reply:
> and nest playground settings under settings.

I'm not sure what you mean here. Where is "settings".? (There's a lot of things called settings)

Reply:
I mean to move from this:
```typescript
export type LocalItemSettings = {
  collapsed: boolean;
} & Pick<
  PartialPlaygroundSettings,
  "distributionChartSettings" | "functionChartSettings"
>;
```

To this:
```typescript
export type LocalItemState = {
  collapsed: boolean;
  calculator?: CalculatorState;
  settings: Pick<
    PartialPlaygroundSettings,
    "distributionChartSettings" | "functionChartSettings"
  >;
}
```

This is exactly for the purpose of avoid the naming confusion.

(this will require many renames in other places, but it's all straightforward)